src/Http/BunqResponse.php

```diff
@@ -40,4 +47,12 @@ public function getHeaders()
     {
         return $this->headers;
     }
+
+    /**
+     * @return Pagination|null
+     */
+    public function getPagination()
```

Comment:
getPaginationOrNull? I think really helps making code breaks less... your call..

Reply:
It is the problem with the absence of proper notion of nullability in PHP. With PHP 7.1 we will have it, and IDE will be able to say "hey, it can be null!". I'd just leave it as is because then with PHP 7.1 this won't need to be changed back.

In Java, for instance, we do not need to say `orNull` because IDEs do it for us :)

Reply:
agreed